src/world.h

```diff
@@ -40,6 +43,18 @@ struct World {		/* World structure */
 
 extern World *defaultworld;
 
+/* Macros to get string field from world or defaultworld */
+#define world_type(w) \
+   (w->type ? w->type : defaultworld ? defaultworld->type : NULL)
+#define world_character(w) \
+   (w->character ? w->character : defaultworld ? defaultworld->character : NULL)
+#define world_pass(w) \
+   (w->pass ? w->pass : defaultworld ? defaultworld->pass : NULL)
+#define world_mfile(w) \
+   (w->mfile ? w->mfile : defaultworld ? defaultworld->mfile : NULL)
```

Comment:
Seems you copy/pasted that second time..

Reply:
The lines look similar, but I don't see a repeat. If there is, but just not included here, should be fine to remove the repeated lines.

Reply:
ah, I see what you're saying, yeah, this got duplicated and can be removed.

Reply:
so can you remove them?